src/page-modules/contact/machineEvents.ts

```diff
@@ -13,7 +14,8 @@ export const machineEvents = {} as
         | 'isAppTicketStorageMode'
         | 'agreesFirstAgreement'
         | 'agreesSecondAgreement'
-        | 'hasInternationalBankAccount';
+        | 'hasInternationalBankAccount'
+        | 'wantsToBeContacted';
     }
   | {
       type: 'UPDATE_FIELD';
```

Comment:
How does this typing work in practice? I get the idea, but almost seems like at this point we could use `update field(field: string, value: any)` 🤔 Would it be better to have one for each type of value?

Reply:
That is a good idea. In another PR, I am looking at splitting machineEvents into globalEvents and local events for each page. I think this will make it easier to get an overview of which events that are happening in each page. 


This is as example of travelGuaranteeEvents
```

const travelGuaranteeLocalEvents = {} as
  | { type: 'TAXI' }
  | { type: 'CAR' }
  | { type: 'OTHER' }
  | {
      type: 'TOGGLE';
      field: 'isIntialAgreementChecked' | 'hasInternationalBankAccount';
    }
  | {
      type: 'UPDATE_FIELD';
      field: 'kilometersDriven';
      value: string;
    }
  | { type: 'SET_STATE_SUBMITTED'; stateSubmitted: string | undefined };

export const travelGuaranteeEvents = {} as
  | typeof travelGuaranteeLocalEvents
  | typeof globelEvents;

```

Reply:
Will look at in another PR.